guides/common/modules/proc_using-IdM-credentials-to-log-in-to-the-Project-Web-UI-with-a-Firefox-browser.adoc

```diff
@@ -0,0 +1,31 @@
+[id="Using_IdM_credentials_to_log_in_to_the_{Project}_Web_UI-with-a-Firefox-browser_{context}"]
+= Using IdM credentials to log in to the {Project} Web UI with a Firefox browser
+
+This section describes how to use the Firefox browser to log in to your {Project} Web UI with your Identity Management (IdM) login + password.
+
+.Prerequisites
+* Your system administrator has enrolled your {Project} server into IdM and configured the server to use IdM for authentication. For more information, see {InstallingServerDocURL}Configuring_External_User_Groups_{project-context}#Using_FreeIPA_admin[Using Red Hat Identity Management] in _{InstallingServerDocTitle}_.
+* An IdM server is running and reachable.
+* The host on which you are using a Firefox browser to log in to the {Project} Web UI is an IdM client.
+* You have a valid IdM login and password.
+* [Recommended] You are using the latest version of the Firefox browser.
```

Comment:
Having the "[Recommended]" prefix is unusual. Maybe reword to "{Team} recommends using the latest stable Firefox browser."?

Reply:
Thank you, updated now.